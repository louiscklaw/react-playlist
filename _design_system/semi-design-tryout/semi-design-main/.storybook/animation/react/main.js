const config = require('../../base/base');

module.exports = {
  ...config,
  "stories": [
    '../../../packages/(semi-animation-react|semi-animation-styled)/_story/*.react.stories.(js|jsx)',
  ],
  typescript: {
    check: false,
    checkOptions: {}
  },
};
