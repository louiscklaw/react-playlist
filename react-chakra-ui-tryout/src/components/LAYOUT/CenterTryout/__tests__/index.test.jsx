import React from 'react';
import { render } from '@testing-library/react';
import renderer from 'react-test-renderer';

import { CenterTryout } from '../';

describe('test CenterTryout tryout', () => {
  test('helloworld', () => {
    const tree = renderer.create(<CenterTryout />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
