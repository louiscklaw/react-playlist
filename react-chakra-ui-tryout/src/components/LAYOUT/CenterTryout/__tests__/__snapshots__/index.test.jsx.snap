// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test CenterTryout tryout helloworld 1`] = `
<div>
  CenterTryout
  <div>
    <div
      className="css-1f4kpne"
    >
      This is the Center
    </div>
  </div>
  <div>
    <div
      className="chakra-stack css-84zodg"
    >
      <div
        className="css-1gosyp4"
      />
      <div
        className="css-1gosyp4"
      >
        <span
          className="css-v9dupr"
        >
          1
        </span>
      </div>
    </div>
  </div>
  <div>
    <div
      className="chakra-stack css-84zodg"
    >
      <div
        className="css-1kc2j1u"
      />
      <div
        className="css-yh3vrj"
      />
    </div>
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
  <div>
     
  </div>
</div>
`;
